mon-pix/app/components/resume-campaign-banner.js

```diff
@@ -1,29 +1,40 @@
 import _maxBy from 'lodash/maxBy';
 import Component from '@glimmer/component';
 import ENV from 'mon-pix/config/environment';
+import { inject as service } from '@ember/service';
 
 export default class ResumeCampaignBanner extends Component {
 
-  get unsharedCampaignParticipations() {
-    return this.args.campaignParticipations.filter((campaignParticipation) => campaignParticipation.isShared === false);
+  @service currentUser;
+
+  get notFinishedCampaignParticipations() {
+    return this.args.campaignParticipations.filter((campaignParticipation) => !campaignParticipation.assessment.get('isCompleted'));
   }
 
-  get lastUnsharedCampaignParticipation() {
-    return _maxBy(this.unsharedCampaignParticipations, 'createdAt');
+  get lastNotFinishedCampaignParticipation() {
+    return _maxBy(this.notFinishedCampaignParticipations, 'createdAt');
   }
 
   get showResumeBar() {
-    return ENV.APP.IS_PIX_CONTEST !== 'true' ||
-      (ENV.APP.IS_PIX_CONTEST === 'true' && !this.campaignParticipationState.assessment.get('isCompleted'));
+    if (ENV.APP.IS_PIX_CONTEST !== 'true') {
+      return true;
+    }
+    if (this.currentUser.user.finishedPixContestAt) {
+      return false;
+    }
+    if (ENV.APP.IS_PIX_CONTEST === 'true' && !this.campaignParticipationState.assessment.get('isCompleted')) {
```

Comment:
La condition `ENV.APP.IS_PIX_CONTEST === 'true'` est toujours valide à ce niveau
```suggestion
    if (!this.campaignParticipationState.assessment.get('isCompleted'))
```

Reply:
Merci ! Pris en compte !